db/migrate/20231129095416_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.integer :PostCounter
```

Comment:
- [ ] Please make sure to follow the Ruby Naming Convention which dictates that variables are named where all letters are lowercase and words are separated by underscores. Classes and modules use MixedCase and have no underscores, each word starts with a uppercase letter. 🙏 This is to enhance readability and to prevent Ruby from misreading the tables' columns and the class names.

_Please apply those changes in all similar situations._

Reply:
@MahmoudBakr23 Please kindly specify the exact place where these naming conventions are not followed because to my own understanding, I followed the Ruby Naming convention?

Reply:
@Megagig The column names should be :name, :photo, :bio, :posts_counter 👍

Reply:
@MahmoudBakr23 I have implemented these changes. Thank you